<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QCamera</class>
 <widget class="QGroupBox" name="QCamera">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>285</width>
    <height>355</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GroupBox</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="lx_darkness_limit">
     <property name="text">
      <string>Sun altitude limit</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QDoubleSpinBox" name="dsb_darkness_limit">
     <property name="toolTip">
      <string>darkness limit for observation</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="showGroupSeparator" stdset="0">
      <bool>true</bool>
     </property>
     <property name="suffix">
      <string>Â°</string>
     </property>
     <property name="decimals">
      <number>1</number>
     </property>
     <property name="minimum">
      <double>-18.000000000000000</double>
     </property>
     <property name="maximum">
      <double>0.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.500000000000000</double>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <layout class="QHBoxLayout" name="hl_station_info">
     <item>
      <widget class="QPushButton" name="bt_apply">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="toolTip">
        <string>save the changes</string>
       </property>
       <property name="text">
        <string>Apply</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="bt_discard">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="toolTip">
        <string>reset the changes</string>
       </property>
       <property name="text">
        <string>Discard</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QUfoManager" name="ufo_manager">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>50</height>
      </size>
     </property>
     <property name="title">
      <string>UFO manager</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QScannerBox" name="scanner">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>70</height>
      </size>
     </property>
     <property name="title">
      <string>Watch directory</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="QStorageBox" name="storage_primary">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>70</height>
      </size>
     </property>
     <property name="title">
      <string>Primary storage</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="2">
    <widget class="QStorageBox" name="storage_permanent">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>70</height>
      </size>
     </property>
     <property name="title">
      <string>Permanent storage</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QUfoManager</class>
   <extends>QGroupBox</extends>
   <header>widgets/qufomanager.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QStorageBox</class>
   <extends>QGroupBox</extends>
   <header>widgets/storage/qstoragebox.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QScannerBox</class>
   <extends>QGroupBox</extends>
   <header>widgets/storage/qscannerbox.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
